src/main/kotlin/org/rust/ide/annotator/fixes/AddWrapUnsafeFix.kt

```diff
@@ -0,0 +1,30 @@
+package org.rust.ide.annotator.fixes
+
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import org.rust.lang.core.psi.RsBlock
+import org.rust.lang.core.psi.RsExpr
+import org.rust.lang.core.psi.RsExprStmt
+import org.rust.lang.core.psi.RsPsiFactory
+
+class AddWrapUnsafeFix(val expr: RsExpr) : LocalQuickFixAndIntentionActionOnPsiElement(expr) {
+    override fun getFamilyName() = text
+    override fun getText() = "Add unsafe block around function call"
```

Comment:
I'm unaware of strict guidelines, but names of such fixes usually start with "Surround". I think this one could be named "Surround with unsafe block", so its meaning would be easy to grasp quickly.